steps/07-stage.sh

```diff
@@ -19,33 +21,33 @@ sed "s/#VERSION#/${VERSION:-0.0.0.0}/" <"$PATCHES/PDFiumConfig.cmake" >"$STAGING
 
 cp "$SOURCE/LICENSES" "$STAGING/LICENSE"
 cp "$BUILD/args.gn" "$STAGING"
-cp -R "$SOURCE/public" "$STAGING/include"
+cp -R "$SOURCE/public" -T "$STAGING/include"
```

Comment:
Without `-T`, the `public/` directory would be moved into `include/` if it already exists.
While this is irrelevant under CI circumstances, it's a bit confusing for local testing.